oracle-explorer-client/src/main/scala/org/bitcoins/explorer/client/SbExplorerClient.scala

```diff
@@ -0,0 +1,123 @@
+package org.bitcoins.explorer.client
+
+import akka.actor.ActorSystem
+import akka.http.scaladsl.model.{
+  ContentTypes,
+  HttpEntity,
+  HttpMethods,
+  HttpRequest,
+  Uri
+}
+import akka.http.scaladsl.{Http, HttpExt}
+import akka.util.ByteString
+import org.bitcoins.crypto.Sha256Digest
+import org.bitcoins.explorer.env.ExplorerEnv
+import org.bitcoins.explorer.model.{
+  CreateAnnouncementExplorer,
+  CreateAttestations,
+  SbAnnouncementEvent
+}
+import org.bitcoins.explorer.picklers.ExplorerPicklers
+import play.api.libs.json.{JsError, JsSuccess, JsValue, Json}
+
+import scala.concurrent.Future
+
+/** A class that implements the Suredbits oracle explorer API */
+case class SbExplorerClient(env: ExplorerEnv)(implicit system: ActorSystem) {
+  import ExplorerPicklers._
+  import system.dispatcher
+  private val httpClient: HttpExt = Http(system)
+
+  /** Lists all events on oracle explorer
+    * @see https://gist.github.com/Christewart/a9e55d9ba582ac9a5ceffa96db9d7e1f#list-all-events
+    * @return
+    */
+  def listEvents(): Future[Vector[SbAnnouncementEvent]] = {
+    val base = env.baseUri
+    val uri = Uri(base + "events")
```

Comment:
All of these require that the `base` ends with a `/`, should put some check that it does